### DISCUSSION TITLE
利用AI發現新藥物

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#233] [General]
![image](https://user-images.githubusercontent.com/91121500/147631725-61e2d1c1-b400-40c8-9651-8c1b9ce43ebd.png)

AI(具體一點：卷積神經網路）可以預測分子與生物蛋白質之間的相互作用，找出未開發的潛在治療方法。

2015年，一家名為Atomwise公司推出了第一個用於藥物設計的深度學習神經網路AtomNet, 有助於加快新藥物的研發。

資料來源：https://www.atomwise.com/how-we-do-it/

**Comment by EricAgumonJou** [comment]
AtomNet系統可在化學作用的三維空間上進行訓練，類似於圖像辨識和網路學習如何在小的空間上讓相似的特徵組合成較大的複雜結構（組合物）。

**Comment by 107602505** [comment]
感覺AI應用到藥物上會具有很大的挑戰性，畢竟藥物是要使用在人或是動物身上，雖然目的是能夠加速藥物的開發降低成本，但執行方面還是有些缺陷，這篇報導有提到有興趣可以看看~
https://www.thenewslens.com/article/134956

**Reply by 109802503gloria** [reply]
感謝同學提供的參考資料，裡面講的很詳細。
化學分子的結構和生理環境的交互作用真的不是可以簡單地歸納就做成模型的。

**Comment by enoch-lightning** [comment]
因為AI是黑箱作業，而如開發藥品這種要使用如有機化學與藥物動力學的專業知識，不知道使用AI能否超越直接由專業知識所建立出來的模擬模型。
